README.md

```diff
@@ -15,6 +15,8 @@ Most modern Linux distros provide a way to install them via the native package
 manager. Ubuntu 18.04+ is known to ship recent enough versions, older distros do not
 and installing from source may be necessary.
 
+On Windows, just double-click on `build.bat`. It will manage everything automatically.
+
 Dependencies on Windows are best managed using [vcpkg](https://github.com/Microsoft/vcpkg), you can
 find instructions on how to build it on the official repository. Once vcpkg is installed, you should
 install the following packages `physfs glm libsquish` by running
```

Comment:
Wait, do we actually still depend on glm? That was a dependency of REGoth, but I got rid of it in ZenLib, or so I thought.

Reply:
Oh really? Then we might not need it anymore... I should probably update the ZenLib cmake